README.md

```diff
@@ -1,25 +1,7 @@
-# Rsync - Docker mod for openssh-server
+# pnpm - Docker mod for code-server
 
-This mod adds rsync to openssh-server, to be installed/updated during container start.
+This mod adds [PNPM](https://github.com/pnpm/pnpm) to [linuxserver/code-server](https://github.com/linuxserver/docker-code-server), to be installed during container start.
```

Comment:
It should list openvscode-server as well, like here: https://github.com/linuxserver/docker-mods/tree/code-server-flutter?tab=readme-ov-file#flutter-beta---docker-mod-for-code-server-and-openvscode-server